hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -577,9 +586,11 @@ public void validateContainerCommand(
   /**
    * If the container usage reaches the close threshold or the container is
    * marked unhealthy we send Close ContainerAction to SCM.
+   *
    * @param container current state of container
+   * @param cmdType request command type
    */
-  private void sendCloseContainerActionIfNeeded(Container container) {
+  private void sendCloseContainerActionIfNeeded(Container container, Type cmdType) throws StorageContainerException {
```

Comment:
Can we add new function to handle volume full case, and let this sendCloseContainerActionIfNeeded handle container related cases, so the code is more readable

Reply:
done